nwbwidgets/ophys/segmentation.py

```diff
@@ -2,117 +2,33 @@
 
 import numpy as np
 from skimage import measure
-
 import ipywidgets as widgets
 import plotly.graph_objects as go
 import plotly.express as px
-
 from pynwb.base import NWBDataInterface
-from pynwb.ophys import (
-    RoiResponseSeries,
-    DfOverF,
-    PlaneSegmentation,
-    TwoPhotonSeries,
-    ImageSegmentation,
-)
-
-from tifffile import imread, TiffFile
-from ndx_grayscalevolume import GrayscaleVolume
-
-from .base import df_to_hover_text
-from .timeseries import BaseGroupedTraceWidget
-from .utils.cmaps import linear_transfer_function
-from .utils.dynamictable import infer_categorical_columns
-from .controllers import ProgressBar
+from pynwb.ophys import RoiResponseSeries, DfOverF, PlaneSegmentation, ImageSegmentation
 
-import plotly.express as px
+from ..base import df_to_hover_text
+from ..timeseries import BaseGroupedTraceWidget
+from ..utils.cmaps import linear_transfer_function
+from ..utils.dynamictable import infer_categorical_columns
+from ..controllers import ProgressBar
 
 
 color_wheel = px.colors.qualitative.Dark24
 
 
-class TwoPhotonSeriesWidget(widgets.VBox):
```

Comment:
You took this entire class out? Can you handle all the use cases, e.g. reading from an external file?

Reply:
It's actually just renamed: https://github.com/NeurodataWithoutBorders/nwbwidgets/pull/241/files#diff-5bf5b8acc0948de1db1c627366b4eed9ac2b1965bbaf82ca66dc3170992c3444R13

The exact same structure as before is in place, it just deploys to subclass visualizations in a more organized way to allow better scalability (easier to go in and add a quick feature to a single subclass than pile on more lines in a common class that way)